src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -108,7 +95,7 @@ export interface DropdownProps {
   onChange?(newValue: string): void;
 }
 
-export class Dropdown extends Component<DropdownProps> {
+export class BaseDropdown extends Component<DropdownProps> {
```

Comment:
Maybe we do not want to export the `BaseDropdown`, only the one Dropdown at the very bottom?
Now styleguidist has the tree like this on left side menu (and also mentioning the BaseDropdown on the start go the page):

```
Dropdown
  BaseDropdown
  DropdownItem
```

Reply:
Thanks for spotting that. Just a forgotten relic from moving stuff from components I think. Now removed.